susemanager-utils/susemanager-sls/src/modules/virt_utils.py

```diff
@@ -216,3 +221,23 @@ def vm_definition(uuid):
             if cnx:
                 cnx.close()
         return {}
+
+
+def virt_tuner_templates():
+    """
+    Get the virt-tuner templates names
+    """
+    if virt_tuner:
+        return list(virt_tuner.templates.keys())
```

Comment:
Would it make sense to return `sorted(list(...))` so the output is always sorted? I guess that would be mostly for human consumption, if it's only needed in jinja it's probably not important.

Reply:
makes sense. I'll add that then